docs/data-sources/fgs_quotas.md

```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "FunctionGraph"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_fgs_quotas"
+description: |-
+  Using this data source to query the list of available resource quotas for FunctionGraph service within HuaweiCloud.
+---
+
+# huaweicloud_fgs_quotas
+
+Using this data source to query the list of available resource quotas for FunctionGraph service within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_fgs_quotas" {}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `quotas` - All quotas that match the filter parameters.
+
+  The [quotas](#quotas_struct) structure is documented below.
+
+<a name="quotas_struct"></a>
+The `quotas` block supports:
+
+* `type` - The resource type corresponding to quota.
+
+* `unit` - The unit of usage.
+  
+  -> For `fgs_func_code_size`, the unit is `MB`. In other scenarios, there is no unit.
```

Comment:
It’s hard to read the meaning of `fgs_func_code_size` here. Please list and explain the `type` field enumeration values.

Reply:
done.